comparison between pointer and integer

````
/builddir/build/BUILD/percolator-rel-3-01/src/converters/MSToolkit/MSReader.cpp: In member function 'void MSReader::writeSqlite(const char*, MSObject&, char*)':
/builddir/build/BUILD/percolator-rel-3-01/src/converters/MSToolkit/MSReader.cpp:634:37: error: ISO C++ forbids comparison between pointer and integer [-fpermissive]
       if(m.getHeader().header[i] != '\0')
                                     ^~~~
````

Presumably this should be simply comparing to `0` (or `NULL`) not to a `char`

--- a/src/converters/MSToolkit/MSReader.cpp
+++ b/src/converters/MSToolkit/MSReader.cpp
@@ -628,7 +628,7 @@ void MSReader::writeSqlite(const char* c, MSObject& m, char* sha1Report)
   string instrumentType="=";
   for(int i=0; i<16; i++)
     {
-      if(m.getHeader().header[i] != '\0')
+      if(m.getHeader().header[i] != 0)
 	{
 	  string headerLine = m.getHeader().header[i];
 	  if(headerLine.find("CreationDate") != string::npos)

